packages/xo-web/src/common/tooltip/index.js

```diff
@@ -37,7 +37,9 @@ export class TooltipViewer extends Component {
     return (
       <div
         className={classNames(
-          show ? styles.tooltipEnabled : styles.tooltipDisabled,
+          show && content !== undefined
```

Comment:
Mark `content` as optional in proptypes.

Reply:
`content` wasn't marked as required already.